pipelined/bevy_pbr2/src/render/light.rs

```diff
@@ -420,6 +615,11 @@ pub struct LightMeta {
     pub shadow_view_bind_group: Option<BindGroup>,
 }
 
+pub enum LightEntity {
+    Directional(Entity),
```

Comment:
We should use fields here to help self-document intent: light_entity and face_index

Reply:
Nice, I misremembered the struct syntax and thought it would be more awkward but it works nicely, especially with the... destructuring in match arms. Done.